### DISCUSSION TITLE
How will the unwanted language files not be generated?

### DISCUSSION THREAD

**Discussion by luluxiaoyu** [Kinnara/ModernWpf#454] [General]
How will the unwanted language files not be generated?  
I found that many language files are generated after compiling, is there a way to generate only the language I need?

**Comment by FoxTes** [comment]
Yeap. 
Use property `<SatelliteResourceLanguages>you language</SatelliteResourceLanguages>` in .csproj

**Comment by luluxiaoyu** [comment]
> Yeap. Use property `<SatelliteResourceLanguages>you language</SatelliteResourceLanguages>` in .csproj

I have added the entry but it doesn't seem to work, the framework used is .NET Framework 4.8

**Comment by FoxTes** [comment]
> I have added the entry but it doesn't seem to work, the framework used is .NET Framework 4.8

Show your piece of code.

**Comment by inforzip** [comment]
I also tried and the same happens, all language folders appear after recompile.

**Comment by ShingZhanho** [comment]
I came across the same problem a few days ago as well. I was using .NET Framework 4.8. Yesterday I migrated my app to .NET Core 3.1 and everything just worked. It seems like the problem has nothing to do with ModernWPF itself but it is a bug from Microsoft. See dotnet/sdk#3277

I'm afraid you have to migrate to a more recent version of SDK in order to filter unwanted language files.

**Comment by luluxiaoyu** [comment]
> <SatelliteResourceLanguages>you language</SatelliteResourceLanguages>

It is a problem of .NET framework4.8, I migrated the project to. It succeeded after net core6.0
